packages/app/src/App.tsx

```diff
@@ -45,6 +47,30 @@ function App() {
     return { provenance, actions };
   }, [conf]);
 
+  const { sessionId } = getUrlVars();
+  const { workspace } = useRecoilValue(queryParamAtom);
```

Comment:
Would you not want to add the `sessionId` to the `queryParamAtom`? Then you could do

```
const { sessionId, workspace } = useRecoilValue(queryParamAtom);
```